{"uuid": "3ac85762-a1c8-4147-9414-331f9b78ca60", "children": ["d0683ad5-7d86-4781-b04d-64bc2879b258"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "AttributeError: 'Session' object has no attribute 'testfaied'\n", "trace": "  File \"C:\\Users\\piotr\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\piotr\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\piotr\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 901, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\piotr\\PycharmProjects\\selenium_project_14.09.2023\\page_object_pattern\\tests\\conftest.py\", line 15, in setup\n    before_failed = request.session.testfaied\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1694947197771, "stop": 1694947200027}], "start": 1694947197771, "stop": 1694947200409}